src/index.html

```diff
@@ -1,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+
+  <body>
+    Hello, world!
```

Comment:
The page currently only contains a simple 'Hello, world!' text. Consider wrapping it into semantic HTML tag. ✍️